ql/src/java/org/apache/hadoop/hive/ql/stats/BasicStatsNoJobTask.java

```diff
@@ -312,23 +384,23 @@ private int aggregateStats(ExecutorService threadPool, Hive db) {
     return ret;
   }
 
-  private int updatePartitions(Hive db, List<FooterStatCollector> scs, Table table) throws InvalidOperationException, HiveException {
+  private int updatePartitions(Hive db, List<StatCollector> scs, Table table) throws InvalidOperationException, HiveException {
 
     String tableFullName = table.getFullyQualifiedName();
 
     if (scs.isEmpty()) {
       return 0;
     }
     if (work.isStatsReliable()) {
```

Comment:
note: it might make sense to somehow communicate this `work.isStatsReliable` somehow to the `StatCollector` so it can make that `LOG` entry if it has to...